### ISSUE TITLE
Finalize User Documentation for the "Broker" Workflow

### ISSUE THREAD

**Issue by xjrc** [LLNL-Collaboration/uiuc2015#29]
In order to ensure that the development transition from UIUC to LLNL is as smooth as possible, the existing project documentation should be updated to reflect the most up-to-date features of the project.  Additionally, the documentation should be updated to include a quick start guide, which should outline all of the steps required to set up the "broker" workflow for a first-time user.  One final thing that would be great to include in the documentation is a list of potential future developments that could be added to improve any of the components of the existing implementation.

Once this task is complete, the documentation should have the following information:
- [ ] An up-to-date overview of the "broker" workflow, it's features, and how it can be used.
- [ ] A quick start guide that enumerates the set up steps for first time users.
- [ ] A future work section that outlines improvements that could be made to the existing project implementation.
